matlab/README.md

```diff
@@ -53,10 +76,12 @@ $ cd arrow/matlab
 To build the MATLAB interface, use [CMake](https://cmake.org/cmake/help/latest/):
 
 ```console
-$ cmake -S . -B build 
+$ cmake -S . -B -D MATLAB_ARROW_INTERFACE=ON build 
```

Comment:
```suggestion
$ cmake -S . -B build -D MATLAB_ARROW_INTERFACE=ON
```

Reply:
Thanks for catching this typo, @kou! Sorry for making this mistake!

Reply:
No problem. :-)